benchmark/sparse/linear_regression/run_sparse_benchmark.py

```diff
@@ -0,0 +1,40 @@
+"""
+Prepare data for running benchmark on sparse linear regression model
+"""
+
+import argparse
+
+import keras_tf_model
+import mxnet as mx
+import mxnet_sparse_model
+from scipy import sparse
+
+
+def invoke_benchmark(batch_size, epochs):
```

Comment:
Nit: it would be useful going forward if you have 2 function invoke_tf_keras_benchmark() and invoke_mxnet_benchmark(). and from CLI take an option for framework name may be?

Reply:
Yes that was in the earlier revisions - we decided to just go forward with having only one function with invoking both the benchmarks.